db/migrate/20160523202611_create_profiles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateProfiles < ActiveRecord::Migration
+  def change
+    create_table :profiles do |t|
+
+      t.integer :user_id, index: true
+      t.string  :user_full_name
+      t.string  :user_email
+      t.string  :user_phone
+      t.datetime :created_at, null: false
+      t.datetime :updated_at, null: false
+    end
+  end
+end
```

Comment:
add some more fields to profile. Inspect other web-sites(or google it) to have at least couple of extra fields